Resources/_template_week2_Haikus.md

```diff
@@ -0,0 +1,31 @@
+# Template for the haiku assignment
+
+## Instructions
+
+1. Form a subteam of three students from your team.
+2. As a subteam, create one branch on GitLab named `Group<groupnumber>/Haiku_<teamletter><subteamnumber>` (e.g. `Group1/Haiku_A1`).
+3. Copy this Template and rename it to the same name as the branch; keep the extension `.md` (e.g. `Group1/Haiku_A1.md`).
+4. One of the teammembers writes the first line of the haiku in the file on their local computer. Then uploads it to the branch.
+5. The next teammember pulls and edits the file, writing the second line of the haiku in the file. Then uploads it to the branch.
+6. The last teammember pulls and edits the file, and writing the third line of the haiku in the file. Then uploads it to the branch.
+7. One of the students creates a Merge Request to merge the file with the haiku into the master branch.
+8. The subteam assigns the other students from the team (but not from your subteam) to review the Merge Request.
+9. Once the reviewers have confirmed that it is a valid haiku, they can approve the Merge Request.
+10. One of the students from the subteam who wrote the haiku can merge the Merge Request.
+11. Once all haikus in the group folder are merged, the students will read all haiku poems aloud and the best one will be selected by the group.
+
+You are _not_ allowed to use Generative AI for this assignment.
+
+### Rules for a valid haiku
+A haiku is a Japanese poetic form that consists of three lines
+1. five syllables in the first line
+2. seven syllables in the second line
+3. five syllables in the third line
+
+The haiku should be about Open Science.
+
+## The haiku
+_Use this space to write your haiku about Open Science. Each student should contribute one line._
+
+## The team
+_Use this space to list the members of your team (using `@<studentnumber>` tags)._
```

Comment:
I think it is better to work with git.science accounts, which are based on solisID. for most students this will be the same as student number, but not all.